resources/management/commands/resources_spaces_import.py

```diff
@@ -0,0 +1,79 @@
+import csv
+import os
+
+from django.core.management import BaseCommand, CommandError
+from django.utils import translation
+
+from resources.models import Resource, Unit, ResourceType, Purpose, ResourceGroup
+
+
+class Command(BaseCommand):
+    help = "Import resources via csv file. This is for importing resources for spaces"
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "file_path", help="Path to the csv file",
+        )
+
+    @staticmethod
+    def get_authentication_type(authentication_type_fi):
+        if authentication_type_fi == 'ei mitään':
+            return 'none'
+        with translation.override("fi"):
+            name = [k for k in dict(Resource.AUTHENTICATION_TYPES).keys() if
+                    dict(Resource.AUTHENTICATION_TYPES)[k] == authentication_type_fi]
+            return name[0]
+
+    def handle(self, *args, **options):
+        file_path = options["file_path"]
+        if not os.path.exists(file_path):
+            raise CommandError("File {0} does not exist".format(file_path))
+
+        with open(file_path) as f:
+            csv_reader = csv.reader(f, delimiter=';')
+            print('Processing...')
+            for idx, row in enumerate(csv_reader):
+                if idx == 0 or not row[4]:
```

Comment:
Would be nice if the column indexes would be named in a dict (or enum/const) so that they would be accessible like `row[columns["title"]]` (dict)  or `row[COLUMNS.TITLE]` (enum/const)

Reply:
I'll check it